### PR TITLE
Cloudformation plugin 2.1.8

### PR DESCRIPTION
**Pull request by pethers**
Add latest release

### CODE REVIEW

File: cloudformation.properties [hunk 1]
```diff
@@ -2,12 +2,18 @@ name=Cloudformation
...
-publicVersions=2.0.10
+archivedVersions=2.0.1,2.0.2,2.0.3,2.0.4,2.0.5,2.0.7,2.0.8,2.0.9,2.1.18
+publicVersions=2.1.18
 
 defaults.mavenGroupId=com.hack23.sonar
 defaults.mavenArtifactId=sonar-cloudformation-plugin
 
+2.1.18.description=Adds support for checkov. Include rules for terraform, kubernetes and azure templates.
+2.1.18.date=2021-11-09
+2.1.18.sqVersions=[7.9,8.9.*]
```
Comment (ganncamp) [inline cloudformation.properties:13]: All support for 7.9.* is officially over. This range needs to start at 8.9. And do you really mean to _end_ it at 8.9 & not at least support 9.1?
Also, <checks email archives>, if you want to continue supporting Checkov, CNF_NAG &etc reports if there's no conflict with a double-report of metrics. I guess that would be a different version?

File: cloudformation.properties [hunk 2]
```diff
@@ -2,12 +2,18 @@ name=Cloudformation
...
+2.1.18.changelogUrl=https://github.com/Hack23/sonar-cloudformation-plugin/releases/tag/sonar-cloudformation-plugin-2.1.18
```
Comment (ganncamp) [inline cloudformation.properties:14]: 404 for me

File: cloudformation.properties [hunk 3]
```diff
@@ -2,12 +2,18 @@ name=Cloudformation
...
+2.1.18.downloadUrl=https://oss.sonatype.org/service/local/repositories/releases/content/com/hack23/sonar/sonar-cloudformation-plugin/2.1.18/sonar-cloudformation-plugin-2.1.18.jar
```
Comment (ganncamp) [inline cloudformation.properties:15]: also 404

**Review by ganncamp — CHANGES_REQUESTED** [review]

File: cloudformation.properties (hunk 1 above)
Comment (pethers) [reply cloudformation.properties]: Only tested/use it on 8.9 and old LTS. Might work on other versions but need to setup some integration test before I want to say it is supported.

File: cloudformation.properties (hunk 2 above)
Comment (pethers) [reply cloudformation.properties]: Release I made an error, the release was 2.1.8 and not 2.1.18 so did some cut and paste errors. But corrected but links now.

**Review by ganncamp — APPROVED** [review]

**Pull request merged**